enhancements/service-catalog/service-catalog-and-broker-removal.md

```diff
@@ -0,0 +1,266 @@
+1. Remove the Template Service Broker from the OpenShift 4.4 release payload (ART)
+
+#### Story 4
+
+Remove the Brokers from productization
+
+1. Remove the Template Service Broker from productization (ART)
+1. Remove the Ansible Service Broker from productization (ART)
+1. Remove the broker sections from the documentation (Docs)
+
+#### Story 5
+
+Remove the broker operators from productization
+
+1. Remove the Template Service Broker Operator from productization (ART)
+1. Remove the Ansible Service Broker Operator from productization (ART)
+1. Remove the operator instructions from the documentation (Docs)
+
+#### Story 6
+
+1. Remove the Service Catalog from productization (ART)
+1. Remove the installation instructions from the documentation (Docs)
+1. Ensure the how to disable service catalog instructions are correct (Docs/Eng)
+
+### Implementation Details/Notes/Constraints [optional]
+
+N/A
+
+### Risks and Mitigations
+
+N/A
+
+## Design Details
+
+### Test Plan
+
+#### cluster-svcat-apiserver-operator
+ - add tests to verify apiservice and associated objects are removed
+
+#### cluster-svcat-controller-manager-operator
+ - add tests to verify controller-manager and associated objects are removed
+
+#### Upgrades
+ - upgrade testing will be crucial for this feature
+ - if there are tests written for upgrades, we will want to add to them
+
+### Graduation Criteria
+
+The Service Catalog and Brokers have reached the removal stage of the graduation
+criteria.
+
+##### Removing a deprecated feature
+
+- In 4.1, the Service Catalog and Brokers were marked as deprecated in the
+  documentation, see the [release notes](https://docs.openshift.com/container-platform/4.1/release_notes/ocp-4-1-release-notes.html).
+
+- In 4.2, a more detailed announcement of their deprecation was
+  released: [4.2 release notes](https://docs.openshift.com/container-platform/4.2/release_notes/ocp-4-2-release-notes.html#ocp-41-deprecated-features)
+
+- [Support for templates](https://docs.openshift.com/container-platform/4.2/release_notes/ocp-4-2-release-notes.html#ocp-4-2-general-web-console-updates) without using the Template Service Broker and Service
+  Catalog added to OpenShift 4.2.
+
+- In 4.3, we alert users when the Service Catalog or its Brokers are in use.
+
+- In 4.4, Service Catalog and Brokers will be removed from shipping.
+
+### Upgrade / Downgrade Strategy
+
+Clusters will be allowed to upgrade and the operators will remove the Service
+Catalog and Service Brokers during upgrade via a Job.
+
+### Version Skew Strategy
+
+N/A
+
+## Implementation History
+
+- 20190916 - Proposal for removal preparation was written
+- 20191217 - Proposal to remove Service Catalog and Brokers
+- 20191220 - Revised this proposal
+
+## Drawbacks
+
+The idea is to find the best form of an argument why this enhancement should _not_ be implemented.
+
+* what about Custom Broker installed, how will those be handled?
```

Comment:
They stay running they just don't do anything. I think the same should be said for ASB and TSB IMO we should not remove them just orphan them

Reply:
added answer to the doc